module/zfs/arc.c

```diff
@@ -7667,8 +7642,8 @@ arc_init(void)
 		kstat_install(arc_ksp);
 	}
 
-	arc_evict_zthr = zthr_create_timer("arc_evict",
-	    arc_evict_cb_check, arc_evict_cb, NULL, SEC2NSEC(1));
+	arc_evict_zthr = zthr_create("arc_evict",
+	    arc_evict_cb_check, arc_evict_cb, NULL);
```

Comment:
Why was the changed from `zthr_create_timer` to `zthr_create`?  We still want the evict thread to periodically run.

Reply:
The evict thread is run on demand by wakeup. I had previously changed this from `ztrh_create` to `zthr_create_timer` to keep arc tuning updated, but since then the tunables have all been given handlers that call `arc_tuning_update()` as necessary.